### PR TITLE
Update changelog for 1.12.3.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,79 +1,60 @@
 # C/C++ for Visual Studio Code Changelog
 
-## Version 1.12.2 (pre-release): August 24, 2022
-## Enhancements
+## Version 1.12.3: August 29, 2022
+### New Features
+* Enable generate Doxygen comment through context menu, code action and typying. [#5683](https://github.com/microsoft/vscode-cpptools/issues/5683)
```
Comment (michelleangela) [inline Extension/CHANGELOG.md]: Remove "enable" and just say "Generate Doxygen...". "Enable" makes it sound like it was there before but this is a new feature.

**Review by michelleangela — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (michelleangela) [reply Extension/CHANGELOG.md]: Remove "enable" from the enhancements items too.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Oh I see -- the 1.12.1 has the upated comments -- I was using old ones.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: I've re-worded the Enable enhancements as well.

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,79 +1,61 @@
...
 * Fix files with a `.c` extension not using a C++ `std` version passed in the `compilerArgs` or `compilerFragments`. [#9628](https://github.com/microsoft/vscode-cpptools/issues/9628)
 * Fix unnecessary IntelliSense process restarting on file creation handling. [#9630](https://github.com/microsoft/vscode-cpptools/issues/9630)
+* Fix tag parsing of classes and enums with attributes. [#9672](https://github.com/microsoft/vscode-cpptools/issues/9672)
+* Add PID to the extended remote process picker. [PR #9673](https://github.com/microsoft/vscode-cpptools/pull/9673)
+* Fix tag parser crash. [#9679](https://github.com/microsoft/vscode-cpptools/issues/9679), [#9695](https://github.com/microsoft/vscode-cpptools/issues/9695)
+* Fix code analysis fixes generating invalid code when the fix has escaped characters. [#9683](https://github.com/microsoft/vscode-cpptools/issues/9683)
+* Fix unintended generation of `nul.d` file when querying clang or gcc, when compiler arguments include dependency generation arguments. [9707](https://github.com/microsoft/vscode-cpptools/issues/9707)
```
Comment (Colengms) [inline Extension/CHANGELOG.md]: Could you add a "#" here?

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,79 +1,61 @@
...
-* Enable colorized output in cppBuild tasks for clang. [#9643](https://github.com/microsoft/vscode-cpptools/issues/9643)
-* Cache and reuse SSH passwords in the current remote debugging session. [PR #9654](https://github.com/microsoft/vscode-cpptools/pull/9654)
 * Do std fallback when compiler querying, even when explicitly specified via a compiler arg.
 
-## Bug Fixes
-* Fix C++20 IntelliSense parsing bugs. [#7446](https://github.com/microsoft/vscode-cpptools/issues/7446), [#9215](https://github.com/microsoft/vscode-cpptools/issues/9215)
-* Fix crash when tag parsing files containing certain string literals. [#9538](https://github.com/microsoft/vscode-cpptools/issues/9538)
+### Bug Fixes
+* Fix several IntelliSense parsing bugs. [#3683](https://github.com/microsoft/vscode-cpptools/issues/3683), [#6659](https://github.com/microsoft/vscode-cpptools/issues/6659), [#7446](https://github.com/microsoft/vscode-cpptools/issues/7446), [#9215](https://github.com/microsoft/vscode-cpptools/issues/9215)
+* Fix crash when tag parsing files containing certain string literals.  [#9538](https://github.com/microsoft/vscode-cpptools/issues/9538)
```
Comment (Colengms) [inline Extension/CHANGELOG.md]: nit: there's an extra space here.

**Review by Colengms — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
@michelleangela Did you still want changes?

**Review by michelleangela — APPROVED** [review]

**Pull request merged**